src/app/plugins/chanban.plugin.ts

```diff
@@ -50,9 +50,17 @@ export class ChanBanPlugin extends Plugin {
         channel_objs as GuildChannel[]
       );
       if (successfully_banned_channels.length) {
-        message.reply(
-          `Banned user from ${successfully_banned_channels.map((c) => c.name).join(', ')}`
-        );
+        this.container.messageService
+          .sendTextMessage(
+            message,
+            `Banned user from ${successfully_banned_channels.map((c) => c.name).join(', ')}`,
+            { reply: true, delimiter: ' ' }
+          )
+          .catch((err) => {
+            this.container.loggerService.warn(
+              `Failed to inform ${message.author.username} on what channels ${username} was banned from successfully. Error info:\n${err}`
+            );
+          });
```

Comment:
No one is going to get banned in 2000 chars worth of channels. They would just be outright banned before that 😆

Reply:
Sounds like a challenge